docs_src/microservices/security/Dockerfile-remote-sshd

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y openssh-server
+RUN mkdir /var/run/sshd
+RUN echo 'root:THEPASSWORDYOUCREATED' | chpasswd
+RUN sed -i 's/PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
```

Comment:
The authorized_keys file should be injected at container build time instead of allowing passworded root logins.

Reply:
an example...not pursue that at this time..it could be done that way but more complicated